bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/package-info.java

```diff
@@ -34,7 +34,7 @@
  * version, is bound to this proxy component resource type.
  * </p>
  */
-@Version("5.0.0")
+@Version("6.0.0")
```

Comment:
this cannot be merged now.

Reply:
removed semantic version change from af-core, and reduced version to 4.3